avocado/utils/dmesg.py

```diff
@@ -115,3 +122,19 @@ def collect_errors_by_level(output_file=None, level_check=5):
             err += " Please check  dmesg log in debug log."
             LOGGER.debug(dmsg_log)
         raise DmesgError("Test is failed {}".format(err))
+
+
+def skip_dmesg_error(dmesg_errors, skip_dmesg):
+    """Remove dmesg error which want to skip.
+
+    :dmesg_errors dmesg messages
+    :type dmesg_errors: string
```

Comment:
Could you explain better what you are passing here? I see that you are expecting commas (,) in that string. Is a list of lines separated by , ?

Reply:
dmesg_errors is lines of dmesg messages, which is output of dmesg error by level, here not expecting commas.
and expecting commas in skip_dmesg what user will pass for remove.

Reply:
Could you then, explain better this on the docstrings, please?

Reply:
ok